Formula/litani.rb

```diff
@@ -25,11 +26,6 @@ class Litani < Formula
     sha256 "594c67807fb16238b30c44bdf74f36c02cdf22d1c8cda91ef8a0ed8dabf5620a"
   end
 
-  resource "voluptuous" do
```

Comment:
Just want to make sure you intended to remove this since the only other changes are formatting?